Frontend/ui-customer/src/Flow.purs

```diff
@@ -909,8 +917,19 @@ homeScreenFlow = do
                                         let (RideBookingAPIDetails bookingDetails) = resp.bookingDetails
                                         let (RideBookingDetails contents) = bookingDetails.contents
                                         let (RideAPIEntity ride) = fromMaybe dummyRideAPIEntity (resp.rideList !! 0)
+                                        let (FareBreakupAPIEntity fareBreakup) = fromMaybe HomeScreenData.dummyFareBreakUp (resp.fareBreakup !! 0)
```

Comment:
@ashkriti can we remove this variable if its not being used and also remove the imports for this